apps/web/src/pages/templates/components/LackIntegrationAlert.tsx

```diff
@@ -38,32 +38,31 @@ export function LackIntegrationAlert({
 
   return (
     <>
-      <WarningMessage backgroundColor={alertTypeToMessageBackgroundColor(type)}>
+      <WarningMessage
+        backgroundColor={alertTypeToMessageBackgroundColor(type)}
+        onClick={() => {
+          if (isPrimaryMissing) {
+            openSelectPrimaryIntegrationModal({
+              environmentId: environment?._id,
+              channelType: channelType,
+              onClose: () => {
+                segment.track(TemplateEditorAnalyticsEnum.CONFIGURE_PRIMARY_PROVIDER_BANNER_CLICK);
+              },
+            });
+          } else {
+            openIntegrationsModal(true);
+            segment.track(TemplateEditorAnalyticsEnum.CONFIGURE_PROVIDER_BANNER_CLICK);
+          }
+        }}
+      >
         <Group spacing={12} noWrap>
-          <div>
-            <MissingIcon
-              color={alertTypeToDoubleArrowColor(type)}
-              onClick={() => {
-                if (isPrimaryMissing) {
-                  openSelectPrimaryIntegrationModal({
-                    environmentId: environment?._id,
-                    channelType: channelType,
-                    onClose: () => {
-                      segment.track(TemplateEditorAnalyticsEnum.CONFIGURE_PRIMARY_PROVIDER_BANNER_CLICK);
-                    },
-                  });
-                } else {
-                  openIntegrationsModal(true);
-                  segment.track(TemplateEditorAnalyticsEnum.CONFIGURE_PROVIDER_BANNER_CLICK);
-                }
-              }}
-            />
-          </div>
+          <AlertIcon color={alertTypeToDoubleArrowColor(type)} alertType={type} />
           <Text color={alertTypeToMessageTextColor(type)}>
             {text
               ? text
               : `Please configure or activate a provider instance for the ${stepNames[channelType]} channel to send notifications over this node`}
           </Text>
+          <CircleArrowRight />
```

Comment:
> Also, I believe this arrow will be shown immediately after the text, and will not at the end of the row like it should.

When placing it just after the text, I need to add another flex container to align the items, and I feel like it's redundant since there is already a flex container `Group` which is a parent for `AlertIcons` and `Text`, so I just added `CircleArrowRight` there too.

Please let me know if I am missing something or need to go with a different approach.

Reply:
@rayy40, in the comment below I added a screenshot if your result, as you can see the arrow is not at the end of the row :)

Reply:
@ainouzgali , Ah sorry. I wasn't able to understand you before ;(